### PR TITLE
merge Encounter manager

### PR DESCRIPTION
**Pull request by Spike-Zbb**

### CODE REVIEW

**Review by caterunions — DISMISSED** [review]

File: Assets/Scripts/EncounterScripts/TileEncounterTrigger.cs [hunk 1]
```diff
@@ -6,10 +6,12 @@ public class TileEncounterTrigger : MonoBehaviour
 {
 	[SerializeField]
 	private Encounter _encounter;
+	[SerializeField]
 	private EncounterManager _encounterManager;
 
 	private void OnTriggerEnter2D(Collider2D collision)
 	{
```
Comment (caterunions) [inline Assets/Scripts/EncounterScripts/TileEncounterTrigger.cs:13]: this should check that the player is the one triggering it, not just any collision

**Review by caterunions — CHANGES_REQUESTED** [review]
add player check to tile encounter trigger

File: Assets/Scripts/EncounterScripts/LevelEncounters.cs [hunk 2]
```diff
@@ -6,7 +6,8 @@
 [CreateAssetMenu(fileName = "FloorEncounters", menuName = "FloorEncounters", order = 3)]
 public class LevelEncounters : ScriptableObject
 {
-	[SerializeField] private List<Encounter> levelEncounters;
+	[SerializeField] 
+	private List<Encounter> levelEncounters;
```
Comment (AlexQuennelle) [inline Assets/Scripts/EncounterScripts/LevelEncounters.cs:10]: Follow naming convention

**Review by AlexQuennelle — CHANGES_REQUESTED** [review]
Please fix code formatting with visual studio code cleanup

File: Assets/Scripts/EncounterScripts/EncounterManager.cs [hunk 3]
```diff
@@ -9,6 +9,11 @@ public class EncounterManager : MonoBehaviour
 	[SerializeField] 
 	private LevelEncounters _levelEncounters;
 
+	private void Awake()
```
Comment (AlexQuennelle) [inline Assets/Scripts/EncounterScripts/EncounterManager.cs:12]: Why is this being called in Awake?

**Review by AlexQuennelle — APPROVED** [review]

**Review by caterunions — APPROVED** [review]

**Pull request merged**